log_injection/Kandhavel-S_DOC_BOOK/sp.py

```
@@ -0,0 +1,189 @@
+            'place': place,
+            'hospital_name': le_hospital_name.inverse_transform([hospital_name_encoded])[0],
+            'doctor_name': le_doctor_name.inverse_transform([doctor_name_encoded])[0],
+            'specialization': specialization,
+            'qualification': le_qualification.inverse_transform([qualification_encoded])[0]
+        }
+        results.append(result)
+
+    return jsonify(results)
+
+if __name__ == '__main__':
+    app.run(debug=True, host='192.168.137.1', port=5000)
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)